fossunited/api/emailing.py

```
@@ -86,15 +98,107 @@ def add_to_email_group(email_group: str, email: str):
+                    add_to_email_group(event_group.name, email)
+                else:
+                    remove_from_email_group(event_group.name, email)
+
+                if subscribe_to_chapter:
+                    add_to_email_group(chapter_group.name, email)
+                else:
+                    remove_from_email_group(chapter_group.name, email)
+            except frappe.DuplicateEntryError:
+                continue
+
+    except Exception:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)